.github/workflows/ci_cd.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: "PyAnsys documentation style checks"
-        uses: ansys/actions/doc-style@v5
+        uses: ansys/actions/doc-style@v6
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You'll need to adapt vale most probably

Reply:
You got a silent failure: https://github.com/ansys/pyansys-math/actions/runs/8738934103/job/23979064484?pr=314#step:2:178

Reply:
True, I need to modify it. Thanks for catching the issue!